common/reviews/api/ts-command-line.api.md

```diff
@@ -30,7 +30,7 @@ export class CommandLineChoiceListParameter extends CommandLineParameter {
     constructor(definition: ICommandLineChoiceListDefinition);
     readonly alternatives: ReadonlyArray<string>;
     // @override
-    appendToArgList(argList: string[]): void;
+    appendToArgList(argList: string[], wrapValueInQuotesIfNecessary?: boolean): void;
```

Comment:
I think quoting values should ultimately be the responsibility of the entity using the argument list. It's a little weird to have it be the responsibility of the parser when the thing consuming the argument list can easily quote the arguments with a single call to `.map()`.

Reply:
Fair enough.